<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegeltestSave</class>
 <widget class="QDialog" name="RegeltestSave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
       <height>566</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Save Regeltest</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="5">
          <widget class="QWidget" name="widget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
          <widget class="QLineEdit" name="icon_path_edit"/>
      </item>
                  <item>
                      <widget class="QPushButton" name="icon_edit_button">
                          <property name="text">
                              <string>Auswählen</string>
                          </property>
                      </widget>
                  </item>
              </layout>
          </widget>
      </item>
      <item row="22" column="0">
          <spacer name="verticalSpacer">
              <property name="orientation">
                  <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
                  <size>
                      <width>20</width>
                      <height>40</height>
                  </size>
              </property>
          </spacer>
      </item>
      <item row="11" column="2">
          <widget class="QWidget" name="widget_6" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                      <widget class="QLineEdit" name="ppt_edit"/>
                  </item>
                  <item>
                      <widget class="QPushButton" name="ppt_edit_button">
                          <property name="text">
                              <string>Auswählen</string>
                          </property>
                      </widget>
                  </item>
              </layout>
          </widget>
      </item>
      <item row="15" column="2" colspan="4">
          <widget class="QWidget" name="widget_4" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                      <widget class="QLineEdit" name="csv_edit"/>
                  </item>
                  <item>
                      <widget class="QPushButton" name="csv_edit_button">
                          <property name="text">
                              <string>Auswählen</string>
                          </property>
                      </widget>
                  </item>
     </layout>
    </widget>
   </item>
      <item row="21" column="0" colspan="6">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
                  <widget class="QPushButton" name="activate_mc_button">
                      <property name="text">
                          <string>Alle Multiple-Choice aktivieren</string>
                      </property>
                  </widget>
              </item>
              <item>
                  <widget class="QPushButton" name="deactivate_mc_button">
                      <property name="text">
                          <string>Alle Multiple-Choice deaktivieren</string>
                      </property>
                  </widget>
              </item>
          </layout>
      </item>
      <item row="6" column="3">
          <widget class="QLabel" name="label_4">
              <property name="text">
                  <string>Schriftgröße</string>
              </property>
          </widget>
      </item>
      <item row="1" column="3">
          <widget class="QLabel" name="label_2">
              <property name="text">
                  <string>Iconpfad</string>
              </property>
          </widget>
      </item>
      <item row="14" column="0">
          <widget class="QLabel" name="label_10">
              <property name="text">
                  <string>Advanced: CSV</string>
              </property>
          </widget>
      </item>
      <item row="13" column="0" colspan="6">
          <widget class="Line" name="line">
              <property name="orientation">
                  <enum>Qt::Horizontal</enum>
              </property>
          </widget>
      </item>
      <item row="6" column="0">
          <widget class="QLabel" name="label_3">
              <property name="text">
                  <string>Speicherort</string>
              </property>
          </widget>
      </item>
      <item row="1" column="2">
          <widget class="QWidget" name="widget_3" native="true">
              <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                      <widget class="QLineEdit" name="title_edit"/>
                  </item>
              </layout>
          </widget>
      </item>
      <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="label_7">
              <property name="text">
                  <string>Allgemeines</string>
              </property>
          </widget>
      </item>
      <item row="17" column="0">
          <widget class="QLabel" name="label_13">
              <property name="text">
                  <string>Fragen-Setup</string>
              </property>
          </widget>
      </item>
      <item row="12" column="2">
          <widget class="QWidget" name="widget_8" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                      <widget class="QSpinBox" name="spinBox_ppt_time">
                          <property name="suffix">
                              <string>Sekunden</string>
                          </property>
                          <property name="minimum">
                              <number>1</number>
                          </property>
                          <property name="maximum">
                              <number>280</number>
                          </property>
                          <property name="value">
                              <number>60</number>
                          </property>
                      </widget>
                  </item>
              </layout>
          </widget>
      </item>
      <item row="1" column="0">
          <widget class="QLabel" name="label">
              <property name="text">
                  <string>Titel</string>
              </property>
          </widget>
      </item>
      <item row="11" column="3">
          <widget class="QLabel" name="label_11">
              <property name="text">
                  <string>Gruppen</string>
              </property>
          </widget>
      </item>
      <item row="8" column="0" colspan="6">
          <widget class="Line" name="line_2">
              <property name="orientation">
                  <enum>Qt::Horizontal</enum>
              </property>
          </widget>
      </item>
      <item row="23" column="0" colspan="6">
          <widget class="QScrollArea" name="scrollArea">
              <property name="widgetResizable">
                  <bool>true</bool>
              </property>
              <widget class="QWidget" name="question_scrollable">
                  <property name="geometry">
                      <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>740</width>
                          <height>69</height>
                      </rect>
                  </property>
              </widget>
          </widget>
      </item>
      <item row="6" column="2">
          <widget class="QWidget" name="widget_2" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                      <widget class="QLineEdit" name="pdf_edit"/>
                  </item>
                  <item>
                      <widget class="QPushButton" name="pdf_edit_button">
                          <property name="text">
                              <string>Auswählen</string>
                          </property>
                      </widget>
                  </item>
              </layout>
          </widget>
      </item>
      <item row="6" column="5">
          <widget class="QWidget" name="widget_5" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                      <widget class="QSpinBox" name="fontsize_spinBox">
                          <property name="suffix">
                              <string>pt</string>
                          </property>
                          <property name="minimum">
                              <number>9</number>
                          </property>
                          <property name="maximum">
                              <number>15</number>
                          </property>
                          <property name="value">
                              <number>11</number>
                          </property>
                      </widget>
                  </item>
              </layout>
          </widget>
      </item>
      <item row="3" column="0" colspan="6">
          <widget class="Line" name="line_3">
              <property name="orientation">
                  <enum>Qt::Horizontal</enum>
              </property>
          </widget>
      </item>
      <item row="24" column="0" colspan="6">
          <widget class="QDialogButtonBox" name="buttonBox">
              <property name="orientation">
                  <enum>Qt::Horizontal</enum>
              </property>
              <property name="standardButtons">
                  <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
              </property>
          </widget>
      </item>
      <item row="12" column="0">
          <widget class="QLabel" name="label_12">
              <property name="text">
                  <string>Zeit pro Frage</string>
              </property>
          </widget>
      </item>
      <item row="11" column="0">
          <widget class="QLabel" name="label_6">
              <property name="text">
                  <string>Speicherort</string>
              </property>
          </widget>
      </item>
      <item row="15" column="0">
          <widget class="QLabel" name="label_5">
              <property name="text">
                  <string>Speicherort</string>
              </property>
          </widget>
      </item>
      <item row="10" column="0" colspan="6">
          <widget class="QLabel" name="label_9">
              <property name="text">
                  <string>Powerpoint Regeltest</string>
              </property>
          </widget>
      </item>
      <item row="16" column="0" colspan="6">
          <widget class="Line" name="line_4">
              <property name="orientation">
                  <enum>Qt::Horizontal</enum>
              </property>
          </widget>
      </item>
      <item row="11" column="5">
          <widget class="QWidget" name="widget_7" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                      <widget class="QSpinBox" name="spinBox_ppt_groups">
                          <property name="minimum">
                              <number>1</number>
                          </property>
                          <property name="maximum">
                              <number>2</number>
                          </property>
                          <property name="value">
                              <number>1</number>
                          </property>
                      </widget>
                  </item>
              </layout>
          </widget>
      </item>
      <item row="4" column="0" colspan="6">
          <widget class="QLabel" name="label_8">
              <property name="text">
                  <string>PDF Regeltest</string>
              </property>
          </widget>
      </item>
      <item row="2" column="0" colspan="6">
          <widget class="QCheckBox" name="regeltest_archive_checkBox">
              <property name="text">
                  <string>Regeltest zusätzlich im Archiv speichern?</string>
              </property>
              <property name="checked">
                  <bool>true</bool>
              </property>
          </widget>
      </item>
  </layout>
 </widget>
    <tabstops>
        <tabstop>title_edit</tabstop>
        <tabstop>icon_path_edit</tabstop>
        <tabstop>icon_edit_button</tabstop>
        <tabstop>pdf_edit</tabstop>
        <tabstop>pdf_edit_button</tabstop>
        <tabstop>widget_3</tabstop>
    </tabstops>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>RegeltestSave</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RegeltestSave</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RegeltestSave</sender>
   <signal>windowTitleChanged(QString)</signal>
   <receiver>title_edit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
